Description: Avoid conflicting (and unused) bswap_dbl definition from ffmpeg.
Author: Adam Conrad <adconrad@ubuntu.com>

--- mythtv-0.27.1+fixes.20140624.aa822f5.orig/mythtv/external/FFmpeg/libavutil/bswap.h
+++ mythtv-0.27.1+fixes.20140624.aa822f5/mythtv/external/FFmpeg/libavutil/bswap.h
@@ -77,7 +77,7 @@ static inline uint64_t av_const av_bswap
 #endif
 
 #ifndef av_bswap_dbl
-static inline double bswap_dbl(double x)
+static inline double av_bswap_dbl(double x)
 {
     union {
         uint32_t l[2];
